py/examples/image.py

```diff
@@ -26,11 +26,11 @@
         0.76648938, 0.89679732, 0.77222302, 0.92717429, 0.61465203, 0.60906377,
         0.68468487, 0.25101297, 0.83783764, 0.11861562, 0.79723474, 0.94900427,
         0.14806288])) ** 2,
-    c=[0.90687198, 0.78837333, 0.76840584, 0.59849648, 0.44214562, 0.72303802,
-       0.41661825, 0.2268104, 0.45422734, 0.84794375, 0.93665595, 0.95603618,
-       0.39209432, 0.70832467, 0.12951583, 0.35379639, 0.40427152, 0.6485339,
-       0.03307097, 0.53800936, 0.13171312, 0.52093493, 0.10248479, 0.15798038,
-       0.92002965],
+    c=[(0.90687198, 0.0, 0.0), (0.78837333, 0.0, 0.0), (0.76840584, 0.0, 0.0), (0.59849648, 0.0, 0.0), (0.44214562, 0.0, 0.0), (0.72303802, 0.0, 0.0),
+        (0.41661825, 0.0, 0.0), (0.2268104, 0.0, 0.0), (0.45422734, 0.0, 0.0), (0.84794375, 0.0, 0.0), (0.93665595, 0.0, 0.0), (0.95603618, 0.0, 0.0),
+        (0.39209432, 0.0, 0.0), (0.70832467, 0.0, 0.0), (0.12951583, 0.0, 0.0), (0.35379639, 0.0, 0.0), (0.40427152, 0.0, 0.0), (0.6485339, 0.0, 0.0),
+        (0.03307097, 0.0, 0.0), (0.53800936, 0.0, 0.0), (0.13171312, 0.0, 0.0), (0.52093493, 0.0, 0.0), (0.10248479, 0.0, 0.0), (0.15798038, 0.0, 0.0),
+        (0.92002965, 0.0, 0.0)],
```

Comment:
What is this for?

Reply:
image.py:29: error: List item 1 has incompatible type "float"; expected "tuple[float, float, float] | str | tuple[float, float, float, float] | tuple[tuple[float, float, float] | str, float] | tuple[tuple[float, float, float, float], float]"  [list-item]

Reply:
I'll work on it to see if there's another way, its currently all red dots..

Reply:
I've looked into this one, in the plt.scatter document said "A scalar or sequence of n numbers to be mapped to colors using cmap and norm." which I think the original code is perfectly fine, but mypy doesn't think so, ill put ignore on it for now, if you want me to put in random value to satisfy the mypy let me know.